### ISSUE TITLE
Autoplay is not working (Youtube, Vimeo)

### ISSUE THREAD

**Issue by OneX** [mcstudios/glightbox#46]
I am invoking plugin with this code.

var lightboxDescription = GLightbox({
  selector: 'fancybox-media',
  autoplayVideos: true
});

Tested in chrome. https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

**Comment by mcstudios** [comment]
Hi @OneX 

Not able the reproduce the error, can you see the same error on the demo site? i understand the new policy change but not sure how it affects youtube and vimeo as GLightbox uses the api provided by those services.

**Comment by OneX** [comment]
Hi @mcstudios, so all modern browsers now block autoplay options, except firefox.

https://help.vimeo.com/hc/en-us/articles/115004485728-Autoplaying-and-looping-embedded-videos.

**Comment by mcstudios** [comment]
Hi @OneX i was fully aware of those new restrictions, what i did not know was how youtube and vimeo will handle that (GLightbox uses only their code) i thought that the api could provide a way to workaround this but in the link you provided i can read "**There is no way to stop these autoplay restrictions**"

As a workaround they mention that you can mute the video, i'm not going to do this but you can, in the demo site you can see how to pass options to vimeo or youtube, there you can set the muted parameter.

So in short, there's no way to workaround this, the only thing we can do is a little change to delegate the permission to cross-origin iframes but still user permission will be required to autoplay in some cases.

**Comment by OneX** [comment]
@mcstudios i will try to use muted option and get back with my findings.

**Comment by mcstudios** [comment]
i'm closing this for now as it's not a problem with GLightbox, as soon as i have some free time i'll dig more on to this new browser restrictions.
